test/images/redis/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.5
+
+RUN apk add --no-cache redis
```

Comment:
Yes, they don't update package versions for older releases except for critical security patches (unlikely for redis): https://pkgs.alpinelinux.org/packages?name=redis&branch=&repo=&arch=&maintainer=